analyze/lib/analyze.js

```diff
@@ -519,6 +519,54 @@ if (0 < ARGUMENTS.length) {
       print(`./cleanup/remove-skeleton-databases.sh <all options you pass to analyze.sh> ${fs.makeAbsolute('skeleton-databases.json')}`);
     }
   };
+  
+  let extractMissingCollections = function (info) {
+    info.missingCollections = [];
+
+    _.each(_.sortBy(info.databases, x => x.name), function (database, name) {
+      let system = database.collections.filter(function (c) {
+        return c.name[0] === '_';
+      }).map(function(c) {
+        return c.name;
+      });
+      let nonSystem = database.collections.filter(function (c) {
+        return c.name[0] !== '_';
+      }).map(function(c) {
+        return c.name;
+      });
+
+      let missing = [];
+      [ "_apps", "_appbundles", "_aqlfunctions", "_graphs", "_jobs", "_queues" ].forEach(function(name) {
+        if (system.indexOf(name) === -1) {
+          missing.push(name);
+        }
+      });
+
+      if (missing.length > 0) {
+        info.missingCollections.push({ database: database.name, missing });
+      }
+    });
+  };
+  
+  let printMissingCollections = function (info) {
+    if (0 < info.missingCollections.length) {
```

Comment:
info.missingCollections.length > 0

Reply:
Seems you dislike [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions).
I wouldn't use them either because I find them harder to read, but the original script I modified already used them. So there are still several more occurrences of them inside the script.